static/templates/filings/candidate.hbs

```diff
@@ -56,14 +56,28 @@
       <td class="panel__term">Ending cash on hand</td>
       <td class="panel__data">{{{currency cash_on_hand_end_period}}}</td>
     </tr>
+    {{#if net_contributions_period}}
     <tr>
       <td class="panel__term">Net contributions</td>
       <td class="panel__data">{{{currency net_contributions_period}}}</td>
     </tr>
+    {{else}}
+    <tr>
+      <td class="panel__term">Net contributions</td>
+      <td class="panel__data">{{{currency net_contributions_cycle_to_date}}}</td>
+    </tr>
+    {{/if}}
+    {{#if net_operation_expenditures_period}}
```

Comment:
Sorry I did not write up that ticket with enough specificity. The change I was talking about was specific to presidential, and not all candidates and it is for the reports rather than totals. 

The new variable is called "net_operating_expenditures_cycle_to_date" and the name of it should be similar. 

It should go in the reports slide out of the filings view:
![screen shot 2015-11-17 at 9 39 00 am](https://cloud.githubusercontent.com/assets/4406333/11214409/139ae1a6-8d0f-11e5-9b32-29c80100d3e0.png)

I will make a PR to this branch.